spyder/config/main.py

```diff
@@ -73,7 +73,11 @@
               'single_instance': True,
               'open_files_port': OPEN_FILES_PORT,
               'tear_off_menus': False,
+              'normal_screen_resolution': True,
               'high_dpi_scaling': False,
+              'high_dpi_custom_scale_factor': False,
+              'high_dpi_custom_scale_factors': '2;',

```

Comment:
Please here a float number, e.g. 1.5 or 2.5, so that people understands that they can adjust the scale factor to anything they want.

Also remove the `;` at the end (which doesn't make sense for a single screen).